config/routes.rb

```diff
@@ -1,20 +1,11 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   root "users#index"
 
   resources :users do
     resources :posts
   end
 
```

Comment:
**[OPTIONA BUT HIGHLY RECOMMENDED]**

- [ ] Great work so far but it would be nice if you have only the necessary endpoints here such as index and show because right now you have all 7 endpoints which are not being used at this time 😨 Kindly specify only those endpoints which are necessary 👍

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```